language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/DocumentServiceKeys.java

```diff
@@ -56,4 +56,6 @@
             = new LanguageServerContext.Key<>();
     public static final LanguageServerContext.Key<String> CURRENT_PACKAGE_NAME_KEY
             = new LanguageServerContext.Key<>();
+    public static final LanguageServerContext.Key<TextDocumentServiceContext> COMPLETION_META_CONTEXT_KEY
```

Comment:
Since we moved this particular key to the DocumentServiceKey class, let's alter the variable name COMPLETION_META_CONTEXT_KEY to OPERATION_META_CONTEXT_KEY